zp-relayer/workers/poolTxWorker.ts

```diff
@@ -78,7 +79,7 @@ export async function createPoolTxWorker<T extends EstimationType>(
 
         await updateNonce(++nonce)
 
-        logger.info(`${logPrefix} TX hash ${txHash}`)
+        jobLogger.info(`TX hash ${txHash}`)
```

Comment:
I think it makes sense to log all tx-related logs with txHash in the properties object under a`txHash` key:
```
jobLogger.info('Sent tx', { txHash })
```